Assignment2/Saabertooth1-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,62 @@
+{
+	"@context": {
+			"el": "http://example.org/elements",
+			"xsd": "http://www.w3.org/2001/XMLSchema",
+			"class": {
+              "@id": "http://example.org/class",
+              "@type": "el:class"
+			},
+			"sensor": {
+              "@id": "http://example.org/sensor",
+			  "@type": "el:sensor"
+			},
+			"computer": {
+              "@id": "http://example.org/computer",
+              "@type": "el:computer"
+			},
+			"measurement": {
+              "@id": "http://example.org/mesurement",
+              "@type": "el:measurement"
+			},
+			"user": {
+              "@id": "http://example.org/user",
+              "@type": "el:user"
+			},
```

Comment:
All of them are objects, thus I suggest you to directly define each type at data definitions out of the @context. E.g you may define "@type":"el:Class", "@type":"el:Sensor", etc. instead of "@type": "class", "@type": "sensor", etc.